components/Basket.tsx

```diff
@@ -43,15 +44,20 @@ export default function Basket() {
 
   const currentStep = Steps[stepIndex];
 
-  const onBackClick = () => setStepIndex(stepIndex - 1);
+  const onBackClick = () => {
+    setStepIndex(stepIndex - 1);
+  };
+
   const onNextClick = () => {
     if (currentStep.onLeave) {
       currentStep.onLeave();
     }
     setStepIndex(stepIndex + 1);
   };
+
   const onOrder = () => {
     dispatch({ type: "CREATE_ORDER", brandId: config.id, client });
+    setIsBusy(true);
```

Comment:
Is it ever set back to false?

Reply:
nope. thats a beauty of matrix. by this single line we force universe to never ever fail on that action!

Reply:
🤖 are you a real person?

Reply:
+[--->++<]>+++.[->++++<]>+.[--->+<]>----.+[++>---<]>.++[--->++<]>.++++[->++<]>+.+[-->+<]>++.---[->+++<]>+.[->+++++<]>-.[->+++<]>+.-[->+++<]>.---[----->++<]>.-------------.----.+++++++++++.[++>---<]>--.-[--->++<]>--.[--->+<]>---.--------.------------.+++++++++++++.-[->+++++<]>-.[->+++<]>++.+++++++++++++.-----------.+[--->+<]>++.

Reply:
hum4n f135h. 0k. n07 5u23 480u7 8241n 7h0u9h.